internal/v2/dtos/requests/secrets_test.go

```diff
@@ -63,13 +65,13 @@ func TestSecretsRequest_Validate(t *testing.T) {
 		Request       SecretsRequest
 		ErrorExpected bool
 	}{
-		{"valid with no path", validNoPath, false},
-		{"valid with with path", validWithPath, false},
-
-		{"invalid no requestId", noRequestId, true},
-		{"invalid no Secrets", noSecrets, true},
-		{"invalid missing secret key", missingSecretKey, true},
-		{"invalid missing secret value", missingSecretValue, true},
+		{"valid - with no path", validNoPath, false},
+		{"valid - with with path", validWithPath, false},
+		{"valid - no requestId", noRequestId, false},
```

Comment:
can make it more consistent with other cases; eg. change `noRequestId` to `validNoRequestId`.

Reply:
Yep, good one! fixed.